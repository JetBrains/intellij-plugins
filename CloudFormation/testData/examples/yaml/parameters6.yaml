AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  MyNestedStack:
    Type: String
    MinValue: 100a
    MaxValue: 1
Resources:
  RootBucket:
    Type: AWS::S3::Bucket
