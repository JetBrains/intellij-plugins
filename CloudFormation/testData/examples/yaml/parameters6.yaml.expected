AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  MyNestedStack:
    Type: String
    0@<1@<MinValue>>: 100a
    2@<MaxValue>: 1
Resources:
  RootBucket:
    Type: AWS::S3::Bucket

0: Expected an integer or float
1: MinValue property is valid for Number type only
2: MaxValue property is valid for Number type only
